xonsh/xontribs.py

```diff
@@ -1,13 +1,13 @@
 """Tools for helping manage xontributions."""
-import os
 import sys
 import json
 import builtins
 import argparse
 import functools
 import importlib
 import importlib.util
-
+import xonsh.xontribs_meta
+import typing as tp
```

Comment:
Typing should go with the standard library imports above.  Xonsh xontribs should be below.

Reply:
updated